notebooks/data-sources/TestGrid/testgrid_metadata_EDA.ipynb

```diff
@@ -283,7 +283,7 @@
      "name": "stdout",
```

Comment:
<p>it looks like its 43 not 52 now</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Reply:
Ah right, forgot the "proofread for reproducibility" :) for these previous sections. I'll update the markdown.

p.s. now that aicoe-aiops/project-template#32 is merged, should we pull changes from the project-template repo so that this repo has the "polish" pull request template too?